synapseclient/models/file.py

```diff
@@ -476,27 +482,27 @@ async def store(
         Example: Using this function
             File with the ID `syn123` at path `path/to/file.txt`:
 
-                file_instance = await File(id="syn123", path="path/to/file.txt").store()
+                file_instance = await File(id="syn123", path="path/to/file.txt").store_async()
```

Comment:
Do we want to update these examples in the docstrings without the async syntax ( `await ...` and `_async()` ) since it's no longer necessary?

Reply:
The non-async method definitions will have the non-async examples: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/f2830f87f72c9468cef8ddccbda569f06e3d012d/synapseclient/models/protocols/file_protocol.py#L51